src/Installer.php

```diff
@@ -218,6 +275,49 @@ private function installPuli()
     }
 
     /**
+     * Validates if the passed version is valid.
+     *
+     * @param string $version  The passed version.
+     * @param array  $versions The available versions.
+     *
+     * @return bool Whether the version is valid.
+     */
+    private function validateVersion($version, array $versions)
+    {
+        return in_array($version, $versions);
```

Comment:
Thanks I didn't know about the `true` argument :) !

Actually, I wanted to change it to the inline version once the `VersionParser` is used.